src/Type/Symfony/InputBagDynamicReturnTypeExtension.php

```diff
@@ -21,14 +25,29 @@ public function getClass(): string
 
 	public function isMethodSupported(MethodReflection $methodReflection): bool
 	{
-		return $methodReflection->getName() === 'get';
+		return in_array($methodReflection->getName(), ['get', 'all'], true);
 	}
 
 	public function getTypeFromMethodCall(
 		MethodReflection $methodReflection,
 		MethodCall $methodCall,
 		Scope $scope
 	): Type
+	{
+		switch ($methodReflection->getName()) {
```

Comment:
I don't like switch for these scenarios, please use a simple `if` statement.

It can also be a totally separate extension as these methods don't have anything in common.

Reply:
I've updated the PR. I took the pattern from `ServiceDynamicReturnTypeExtension`.